### PR TITLE
Add OneFootballCollectible

### PR DESCRIPTION
**Pull request by louisguitton**
```sh
→ flow scripts execute getNFTs.cdc --args-json '[{ "type": "Address", "value": "0x90bd6b4845e310a0" }, { "type": "Dictionary", "value": [{ "key": { "type": "String", "value": "OneFootballCollectible" }, "value": { "type": "Array", "value": [{ "type": "UInt64", "value": "90" }] } }] }]' --network mainnet

⚠️  Version warning: a new version of Flow CLI is available (v0.30.2).
   Read the installation guide for upgrade instructions: https://docs.onflow.org/flow-cli/install


Result: [s.0fce97e69ff4ff364c102076a4fa77fcd12737998f419f73e61f820739d5c89e.NFTData(
  contract: s.0fce97e69ff4ff364c102076a4fa77fcd12737998f419f73e61f820739d5c89e.NFTContract(
    name: "OneFootballCollectible", 
    address: 0x6831760534292098, 
    storage_path: "OneFootballCollectible.CollectionStoragePath", 
    public_path: "OneFootballCollectible.CollectionPublicPath", 
    public_collection_name: "OneFootballCollectible.OneFootballCollectibleCollectionPublic", 
    external_domain: "http://xmas.onefootball.com/"
  ), 
  id: 90, 
  uuid: 104699149, 
  title: "Louis, OneFootballerz #78", 
  description: "OneFootballerz is a collection of personalised jerseys for OneFootball employees.", 
  external_domain_view_url: "https://xmas.onefootball.com/0x90bd6b4845e310a0", 
  token_uri: nil, 
  media: [
    s.0fce97e69ff4ff364c102076a4fa77fcd12737998f419f73e61f820739d5c89e.NFTMedia(
      uri: "https://bafybeif26dqnotrgz5mkbuuj2vsv7wmpii7ymdwu7jhv5e4f55aewitela.ipfs.dweb.link/", 
      mimetype: "video"
    ), 
    s.0fce97e69ff4ff364c102076a4fa77fcd12737998f419f73e61f820739d5c89e.NFTMedia(
      uri: "https://bafkreicbozp5kqoverw5uivz4hfwycb6xnjtxoxsmbegv4554hcg2q6opa.ipfs.dweb.link/", 
      mimetype: "image"
    )
  ], 
  metadata: {
    "player_name": "Louis Guitton", 
    "of_id": "78"
  }
)]
```

### CODE REVIEW

File: src/cadence/mainnet/getNFTs.cdc [hunk 1]
```diff
@@ -1248,4 +1250,45 @@ pub fun getXtinglesNFT(owner: PublicAccount, id: UInt64): NFTData? {
...
+        if let nft = collection.borrowOneFootballCollectible(id: id) {
+            if let metadata = nft.getTemplate() {
+                return NFTData(
+                    contract: contract,
+                    id: nft.id,
+                    uuid: nft.uuid,
+                    title: metadata.name,
+                    description: metadata.description,
+                    external_domain_view_url: "https://xmas.onefootball.com/".concat(owner.address.toString()),
+                    token_uri: nil,
```
Comment (louisguitton) [inline src/cadence/mainnet/getNFTs.cdc:1278]: I'm not sure what this line is for as the Readme doesn't say :/ let me know if I can add anything here @sideninja @rambhask @niveda-krish

File: src/cadence/mainnet/getNFTs.cdc [hunk 2]
```diff
@@ -1248,4 +1250,45 @@ pub fun getXtinglesNFT(owner: PublicAccount, id: UInt64): NFTData? {
...
+                    media: [
+                        // media
+                        NFTMedia(uri: "https://".concat(metadata.media).concat(".ipfs.dweb.link/"), mimetype: "video"),
+                        // preview
+                        NFTMedia(uri: "https://".concat(metadata.preview).concat(".ipfs.dweb.link/"), mimetype: "image")
```
Comment (louisguitton) [inline src/cadence/mainnet/getNFTs.cdc:1283]: also I'm not sure about adding the preview as fallback, as I don't see anyone doing this in this script

File: src/cadence/mainnet/getNFTs.cdc (hunk 1 above)
Comment (niveda-krish) [reply src/cadence/mainnet/getNFTs.cdc:1278]: Token_uri can be used if your metadata is hosted on ipfs/arweave (similar to the way its used in Eth).

File: src/cadence/mainnet/getNFTs.cdc (hunk 2 above)
Comment (niveda-krish) [reply src/cadence/mainnet/getNFTs.cdc:1283]: This looks good

**Comment by niveda-krish** [comment]
Could you resolve merge conflicts from the latest branch and send out a new iteration?

**Review by niveda-krish — APPROVED** [review]

**Comment by louisguitton** [comment]
> Could you resolve merge conflicts from the latest branch and send out a new iteration?

merge conflicts resolved

**Comment by niveda-krish** [comment]
Thanks! So sorry - we just merged some changes today and there are conflicts again - could you merge one more time?

**Comment by louisguitton** [comment]
> Thanks! So sorry - we just merged some changes today and there are conflicts again - could you merge one more time?

Sure, done.

**Comment by louisguitton** [comment]
thanks @niveda-krish for merging this ❤️ 

could you please guide me as to how I can test this change ? how can i see that the NFT is now part of the Alchemy APi?
I've requested access on this form https://www.alchemy.com/nft-api is this what I needed to do?

Thanks in advance 🙏

**Comment by louisguitton** [comment]
hi @niveda-krish can you point me to the right point of contact regarding my question above?

**Pull request merged**